neo/Prompt/Commands/Create.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+from prompt_toolkit import prompt
+
+from neo.Wallets.utils import to_aes_key
+from neo.Prompt.Utils import get_arg
+from neo.Implementations.Wallets.peewee.UserWallet import UserWallet
+from neo.Prompt.CommandBase import CommandBase, SubCommandBase, CommandDesc, ParameterDesc
+from neo.Prompt.PromptData import PromptData
+
+
+class CommandCreate(CommandBase):
```

Comment:
I've always thought that `create wallet` , `open wallet` were confusing when all others commands started with `wallet <something>`. From that point please migrate this to a subcommand of `wallet` to we have it nicely grouped from the start.

Reply:
Moved